pkg/webui/console/store/index.js

```diff
@@ -12,20 +12,88 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+import { mapValues } from 'lodash'
+import * as Sentry from '@sentry/browser'
 import { createStore, applyMiddleware, compose } from 'redux'
 import { createLogicMiddleware } from 'redux-logic'
 import { routerMiddleware } from 'connected-react-router'
+import createSentryMiddleware from 'redux-sentry-middleware'
 
 import dev from '@ttn-lw/lib/dev'
+import env from '@ttn-lw/lib/env'
 
 import createRootReducer from './reducers'
 import requestPromiseMiddleware from './middleware/request-promise-middleware'
 import logics from './middleware/logics'
 
+if (env.sentryDsn)
+  Sentry.init({
+    dsn: env.sentryDsn,
+    release: process.env.VERSION,
+    normalizeDepth: 10,
+  })
+
 const composeEnhancers = (dev && window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__) || compose
 
 export default function(history) {
   const middleware = applyMiddleware(
+    createSentryMiddleware(Sentry, {
+      actionTransformer: action => {
```

Comment:
Please move this to a separate module. I also sense that there might be a library that simplifies scrapping keywords from an object. Can you check that?